api/collections/views.py

```diff
@@ -0,0 +1,283 @@
+from rest_framework import generics, permissions as drf_permissions
+from modularodm import Q
+from datetime import datetime
+
+from framework.auth.core import Auth
+from website.models import Node, Pointer
+from api.base.utils import get_object_or_404
+from api.base.filters import ODMFilterMixin, ListFilterMixin
+from .serializers import CollectionSerializer, CollectionPointersSerializer
+from .permissions import ContributorOrPublic
+from website.views import find_dashboard
+
+smart_folders = {
+    '~amp': Q('category', 'eq', 'project') & Q('is_deleted', 'eq', False) &
```

Comment:
Minor formatting issue, but let's put each Q on a different line. I think that will make it easier to follow with this many criteria.